packages/web-components/src/components/va-select/va-select.scss

```diff
@@ -58,3 +58,7 @@ span.required {
   font-family: var(--font-source-sans);
   padding: 0.8rem 3rem 0.8rem 0.7em;
 }
+
+:host .usa-select {
```

Comment:
We are trying to keep all `v3` specific styles at the top (they go before the comment that says `/** Original Component Style **/`) so do you mind moving those types of styles into that area? Eventually we will need to remove v1 completely and keeping v1 and v3 separate will help us do that.

Also, you don't need to add `:host` here either.